subprojects/gradle/instrumentation-tests/build.gradle.kts

```diff
@@ -7,6 +7,16 @@ plugins {
 }
 
 dependencies {
+    constraints {
+        api(libs.okio) {
+            version { strictly("2.+") }
+            because(
+                "There was an issue dependency-analysis plugin which depends on 1.x version of okio, " +
+                    "caused runtime failures"
+            )
+        }
+    }
```

Comment:
Can we remove workaround for okio from root `build.gradle`?

Reply:
yep, after version release